src/CandidateList.js

```diff
@@ -0,0 +1,105 @@
+import React, {useState, useEffect} from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+function CandidateList() {
+    const [candidates, setCandidates] = useState(null);
+    async function getCandidates() {
+        var data = await axios.get(
+            'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/candidates'
+        );
+        setCandidates(data.data);
+    }
+    function Vote(candidate) {
+        async function UpdateVoteCount() {
+            await axios.get(
+                `http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/vote?id=${candidate.id}` 
+            )
+            .then (function(response){
+                alert(candidate.name + '님에게 투표 완료!');
+            })
+            .catch (function(error) {
+                alert('투표 실패');
+            });
+        }
+        UpdateVoteCount();
+    }
```

Comment:
1. 특별한 이유가 없다면 함수를 중첩할 필요가 없을 것 같아용 

2. `function`키워드로 함수 선언하기 보다 `화살표함수`로 써봐도 간단하고 좋을 것 같습니당 [참고](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Functions/%EC%95%A0%EB%A1%9C%EC%9A%B0_%ED%8E%91%EC%85%98)

3. `handleVoteCount`라는 네이밍도 괜찮을 것 같아요


이렇게 하면 어떨까요??

```suggestion
    const handleVoteCount= async (candidate)=> {
           await axios.get(
                `http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/vote?id=${candidate.id}` 
            )
            .then ((response)=>{
                alert(candidate.name + '님에게 투표 완료!');
            })
            .catch ((error)=> {
                alert('투표 실패');
            });
    }
```

Reply:
감사합니다! 😄